core/src/test/java/org/elasticsearch/search/query/MultiMatchQueryIT.java

```diff
@@ -635,6 +635,37 @@ public void testCrossFieldMode() throws ExecutionException, InterruptedException
         assertFirstHit(searchResponse, hasId("ultimate1"));
     }
 
+    /**
+     * see https://github.com/elastic/elasticsearch/issues/18710
```

Comment:
I'm fine with adding a link in the javadocs but could you add a short explanation so I don't have to go read the link? When I'm scanning javadocs there is 0% chance I'll go and read the link.